webui/react/src/pages/TaskListPage.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import TaskList from 'components/TaskList';
+
+const TaskListPage: React.FC = () => {
+  return <TaskList />;
```

Comment:
moved `TaskList` to components folder since it is not for page anymore. Instead, I make a new component, `TaskListPage`, for the page folder. 
This looks redundant, but i think its more meaningful than pass `<TaskList />` directly in `routes/index.tsx`

Reply:
To cut down on redundancy I do feel like this should remain `TaskList` until there is a reason to throw another wrapper around the component. 

Moving the component to the components folder sounds okay to me.